lesson13-14/home_work13-14.md

```diff
@@ -0,0 +1,517 @@
+dr-xr-xr-x.   5 root root 4096 Dec  4 21:11 boot                        3
+drwxr-xr-x.  20 root root 3080 Dec 19 10:05 dev                         4
+drwxr-xr-x.  74 root root 8192 Dec 19 14:58 etc                         5
+drwxr-xr-x.   4 root root   30 Dec  2 21:16 home                        6
+lrwxrwxrwx.   1 root root    7 Nov 25 19:05 lib -> usr/lib
+lrwxrwxrwx.   1 root root    9 Nov 25 19:05 lib64 -> usr/lib64
+drwxr-xr-x.   2 root root    6 Apr 11  2018 media                       7
+drwxr-xr-x.   2 root root    6 Apr 11  2018 mnt                         8
+drwxr-xr-x.   2 root root   19 Dec 11 02:06 opt                         9
+dr-xr-xr-x. 107 root root    0 Dec 19 10:05 proc                        10
+dr-xr-x---.   5 root root  225 Dec 19 14:03 root                        11
+drwxr-xr-x.  23 root root  700 Dec 19 15:34 run                         12
+lrwxrwxrwx.   1 root root    8 Nov 25 19:05 sbin -> usr/sbin
+drwxr-xr-x.   2 root root    6 Apr 11  2018 srv                         13
+dr-xr-xr-x.  13 root root    0 Dec 19 10:05 sys                         14
+drwxrwxrwt.   9 root root  183 Dec 19 15:34 tmp                         15
+drwxr-xr-x.  13 root root  155 Nov 25 19:05 usr                         16
+drwxr-xr-x.  19 root root  267 Nov 25 19:11 var                         17
+
+```
+
+#### 3. Check what inode numbers have "/" and "/boot" directory. Why?
+```bash
+[sitis@localhost lesson7]$ ls -ali / | grep boot
+      64 dr-xr-xr-x.   5 root root 4096 Dec  4 21:11 boot
+[sitis@localhost lesson7]$ ls -ali /
+total 16
+      64 dr-xr-xr-x.  17 root root  224 Nov 25 19:08 .
+      64 dr-xr-xr-x.  17 root root  224 Nov 25 19:08 ..
+[sitis@localhost lesson7]$ df -Th
+Filesystem              Type      Size  Used Avail Use% Mounted on
+devtmpfs                devtmpfs  908M     0  908M   0% /dev
+tmpfs                   tmpfs     919M     0  919M   0% /dev/shm
+tmpfs                   tmpfs     919M  8.6M  911M   1% /run
+tmpfs                   tmpfs     919M     0  919M   0% /sys/fs/cgroup
+/dev/mapper/centos-root xfs       6.2G  2.0G  4.3G  33% /
+/dev/sda1               xfs      1014M  194M  821M  20% /boot
+tmpfs                   tmpfs     184M     0  184M   0% /run/user/1000
+```
+> Since they are on different filesystems.
+
+#### 4. Check the root directory space usage by du command. Compare it with an information from df. If you find differences, try to find out why it happens.
+```bash
+[sitis@localhost lesson7]$ df -h /
+Filesystem               Size  Used Avail Use% Mounted on
+/dev/mapper/centos-root  6.2G  2.0G  4.3G  33% /
+[sitis@localhost lesson7]$ du -sh /
+2.1G    /
+```
+> The difference is that du queries directly for every file found in the partition, while df queries the filesystem. When a file is deleted, which at that moment was “occupied” by a process, its name is deleted, but the inode remains in the file system until the process that “holds” this file ends.
```

Comment:
Not exactly. Current task is connected with the previous one.

Reply:
@vauboy I didn't quite understand you. Maybe the answer is different number metrics?

Reply:
Could you check du's argument '-x' meaning?

Reply:
I think it will be clear enough to understand, so you can just merge the changes

Reply:
@vauboy I understand correctly now I can merge?